### DISCUSSION TITLE
Tgate issue

### DISCUSSION THREAD

**Discussion by xalteropsx** [huggingface/diffusers#8280] [General]
performance over each steps @standardAI

**Comment by tolgacangoz** [comment]
I will try T-GATE, then turn back here.

**Reply by xalteropsx** [reply]
@standardAI  it solved seems like i forget to disable >.<

**Reply by tolgacangoz** [reply]
I am glad you've done it. [It seems that I couldn't see much difference :thinking:.](https://colab.research.google.com/gist/standardAI/a282db84ad7d00953ce406922bdbe8c6/tgate-deepcache.ipynb)

**Reply by xalteropsx** [reply]
it really strange issue it look like colab work fine  no idea why mine having some strange issue but glad it was fix >.<
